workflow/rules/gatk.smk

```diff
@@ -0,0 +1,151 @@
+__author__ = "Patrik Smeds"
+__copyright__ = "Copyright 2021, Patrik Smeds"
+__email__ = "patrik.smeds@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule gatk_mutect2:
+    input:
+        map="alignment/picard_mark_duplicates/{sample}_{type}_{chr}.bam",
+        bai="alignment/picard_mark_duplicates/{sample}_{type}_{chr}.bam.bai",
+        fasta=config["reference"]["fasta"],
+        bed="snv_indels/bed_split/design_bedfile_{chr}.bed",
+    output:
+        bam=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.bam"),
+        bai=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.bai"),
+        stats=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.vcf.gz.stats"),
+        vcf=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.vcf.gz"),
+        tbi=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.vcf.gz.tbi"),
+        f1f2=temp("snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.f1r2.tar.gz"),
+    params:
+        extra=lambda wildcards: get_mutect2_extra(wildcards, "gatk_mutect2"),
+    log:
+        "snv_indels/gatk_mutect2/{sample}_{type}_{chr}.vcf.gz.log",
```

Comment:
should be adjusted to new output name

Reply:
The vcf-output is called `snv_indels/gatk_mutect2/{sample}_{type}_{chr}.unfiltered.vcf.gz` thus log and benchmark should contain this as a prefix.